src/modules/tests/tests.selector.ts

```diff
@@ -68,3 +68,9 @@ export const isEndToEndPracticeTest = (tests: TestsModel) : boolean =>
 export const getAllTestStatuses = (test: TestsModel): { [slotId: string]: TestStatus; } => {
   return test.testStatus;
 };
+
+export const getActivityCodeBySlotId = (testsModel: TestsModel, id: number): ActivityCode => {
+  if (testsModel && testsModel.startedTests && testsModel.startedTests[id]) {
+    return testsModel.startedTests[id].activityCode;
+  }
+};
```

Comment:
I think it would be better to explicitly return null rather than not returning anything and relying on undefined.

Reply:
Have changed to return null if doesn't meet these conditions